src/api/controllers/schemaController.ts

```diff
@@ -20,6 +20,13 @@ export async function readById(req: Request, res: Response) {
 }
 
 export async function create(req: Request, res: Response) {
+  const { slug, fields } = req.body
+  if (typeof slug !== "string" || !Array.isArray(fields))
```

Comment:
It would be good to separate some of these conditions so we can give more descriptive error messages, e.g. "slug must be a string" or "fields must be an array", rather than a generic "invalid schema format" message.

Reply:
resolved